packages/@ember/-internals/runtime/tests/system/object_proxy_test.js

```diff
@@ -103,6 +103,7 @@ moduleFor(
       if (DEBUG) {
         let proxy = ObjectProxy.extend({
           init() {
+            this._super();
```

Comment:
Because ObjectProxy now correctly extends FrameworkObject, there's a check to ensure `this._super` is called.